### PR TITLE
chore: update contribution

### PR DESCRIPTION
**Pull request by samsja**
# Context

adding our test policy as required by the open SSF badge

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -224,6 +225,14 @@ pip install ".[test]"
 pytest -v -s tests
 ```
 
+### Test policy
+
+Every contribution that add or modify the behavior of a features must include a suit of tests that's validate the feature work as excepted
```
Comment (JoanFM) [inline CONTRIBUTING.md:230]: grammar correction, let's wait for @NicholasDunham to review

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (alaeddine-13) [inline CONTRIBUTING.md:230]:
```suggestion
Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates the feature work as excepted.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -224,6 +225,14 @@ pip install ".[test]"
 pytest -v -s tests
 ```
 
+### Test policy
+
+Every contribution that add or modify the behavior of a features must include a suit of tests that's validate the feature work as excepted 
+
+This allows:
+* reviewer to be very confident that the feature does what it is supposed to do before merging it into the codebase
```
Comment (alaeddine-13) [inline CONTRIBUTING.md:233]:
```suggestion
* the reviewer to be very confident that the feature does what it is supposed to do before merging it into the code base
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -224,6 +225,14 @@ pip install ".[test]"
...
+* contributors to be sure that they don't break already merged features when doing refactoring/modification on the codebase.
```
Comment (alaeddine-13) [inline CONTRIBUTING.md:234]:
```suggestion
* the contributors to be sure that they don't break already merged features when doing refactoring/modification on the code base.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -224,6 +225,14 @@ pip install ".[test]"
 pytest -v -s tests
 ```
 
+### Test policy
+
+Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates the feature work as excepted.
```
Comment (JohannesMessner) [inline CONTRIBUTING.md:230]:
```suggestion
Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates that the feature work as excepted.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -224,6 +225,14 @@ pip install ".[test]"
 pytest -v -s tests
 ```
 
+### Test policy
+
+Every contribution that adds or modifies the behavior of a feature must include a suit of tests that validates that the feature work as excepted.
+
+This allows:
+* the reviewer to be very confident that the feature does what it is supposed to do before merging it into the code base
+* the contributors to be sure that they don't break already merged features when doing refactoring/modification on the code base.
```
Comment (NicholasDunham) [inline CONTRIBUTING.md:234]:
```suggestion
* the contributors to be sure that they don't break already-merged features when refactoring or modifying the code base.
```

**Review by NicholasDunham — APPROVED** [review]
LGTM

**Review by JoanFM — APPROVED** [review]

**Pull request merged**